averages.js

```diff
@@ -0,0 +1,66 @@
+const pry = require('pryjs')
+
+class Operations {
+    constructor(collection) {
+        this.collection = collection
+    }
+
+    get total() {
```

Comment:
I've never seen the `get` syntax before. Cool! That makes it so you can call `operations.total` without `()`, right?